package/com.unity.formats.usd/Documentation~/USD.md

```diff
@@ -1,126 +1,114 @@
-# USD Unity SDK: USD C# Bindings for Unity
+# USD Experimental Package for Unity
 
-This repository contains a set of libraries designed to support the use of
-USD in C#/Unity. The goal of this package is to make it maximally easy to
-integrate and explore Universal Scene Description.
+This package includes a set of libraries designed to support the use of USD in C#, as well as code to import and export USD files into the editor. The goal of this package is to make it maximally easy to integrate and explore Universal Scene Description.
 
-![USD header](Images/USD_header.png)
+![Pixar's USD kitchen Set in the Unity Editor](Images/USD_header.png)
 
-# Getting Started
+*Pixar's [USD Kitchen Set](https://openusd.org/release/dl_kitchen_set.html) in the Unity Editor.*
 
-To get started, install the USD package via the Unity Package manager. You can do this by either:
-* Installing by name, "com.unity.formats.usd";
-* Installing by Git URL;
-* or browsing for a local package while working with
-source.
+## Getting Started
 
-Once the USD package is installed, a USD menu will appear, enabling you to
-easily import and export USD files.
+To get started, install the USD package via the Unity Package Manager. You can do this by either:
+
+* installing by name, "com.unity.formats.usd";
+
+* installing by Git URL, "https://github.com/Unity-Technologies/usd-unity-sdk.git?path=package/com.unity.formats.usd";
+
+* or browsing for a local package while working with source.
+
+Once the USD package is installed, a USD menu will appear, enabling you to easily import and export USD files.
 
 ![USD menu](Images/USD_menu.png)
 
-The USD importer works with linear color space only. To ensure colors are imported correctly,
-set the project color space to "linear" in Edit > Project Settings > Player:
+The USD importer works with linear color space only. To ensure colors are imported correctly, set the project color space to "linear" in Edit > Project Settings > Player:
 
 ![USD linear](Images/USD_linear.png)
```

Comment:
"Images/USD_linear.png" has been deleted in this PR

Reply:
All of the images in the `Images` folder were duplicated in `package/com.unity.formats.usd/Documentation~/Images`, so I've only removed the duplicates at the higher level. This path is relative so *should* use the remaining one.